src/components/ActivityCollector/attachClickActivityCollector.js

```diff
@@ -14,6 +14,10 @@ import { noop } from "../../utils";
 
 const createClickHandler = ({ eventManager, lifecycle, handleError }) => {
   return clickEvent => {
+    // Ignore repropagated clicks from AppMeasurement
+    if (clickEvent.s_fe) {
```

Comment:
AppMeasurement ignores the event that has been "tagged" with `s_fe` because it is the previously captured and cancelled event that was then re-fired. What happened was that Alloy ended up picking up the original event and this event as well causing two link click event getting sent. It might be that this design in AppMeasurement no longer is required (with the beacon API being available etc.). This addition makes it so that alloy is not affected by the "additional " event.

Reply:
That said, depending on the browser prioritization or execution logic for the capturing click handlers this could potentially cause race conditions where we end up losing the event. Perhaps a better solution would be to introduce some sort of rate limiting instead.